scripts/release-notes.py

```diff
@@ -99,6 +107,28 @@ def main(token, repo, num_commits, exclude_dependabot):
 
     if skipped_dependabot:
         print(f"\n(Skipped {skipped_dependabot} dependabot commit{'' if skipped_dependabot == 1 else 's'})")
+     
+    # Print PRs that don't have changelog label 
+    github = Github(token)
+    repository = github.get_repo(f'jaegertracing/{repo}')
+    changelog_prs = []
+    other_prs = []
+    
+    for pr in repository.get_pulls(state='closed'):
+        if 'dependabot' in pr.user.login:
+            continue
+
+        if validate_changelog_labels(pr):
+            changelog_prs.append(pr)
+        else:
+            other_prs.append(pr)
+
+    if other_prs:
+        print('PRs without "changelog:" labels:')
+        for pr in other_prs:
+            print(f'{pr.title} - {pr.html_url}')
```

Comment:
tip: 
  * trim title to fixed length like 50chars
  * use whitespace padding so that URL column always starts from the same place